v2/ticker_service.go

```diff
@@ -102,6 +106,21 @@ func (s *ListPriceChangeStatsService) Symbol(symbol string) *ListPriceChangeStat
 	return s
 }
 
+// Symbols set symbols
+func (s *ListPricesService) Symbols(symbols []*string) *ListPricesService {
+	symbolsParam := `[`
+	for i, symbol := range symbols {
+		symbolsParam = fmt.Sprintf("%v\"%v\"", symbolsParam, *symbol)
+		if i != len(symbols)-1 {
+			symbolsParam = fmt.Sprintf("%v,", symbolsParam)
+		}
+	}
+	symbolsParam = fmt.Sprintf("%v]", symbolsParam)
+
+	s.symbols = &symbolsParam
```

Comment:
It's a setter function, do not change the original value because we need to access it in other functions.
Simply use:
```
 s.symobls = symbols
```